docs/contracts/using-for.rst

```diff
@@ -109,13 +109,13 @@ It is also possible to extend elementary types in that way:
             data.push(value);
         }
 
-        function replace(uint _old, uint _new) public {
+        function replace(uint old, uint new) public {
```

Comment:
"new" is a reserved word as per https://docs.soliditylang.org/en/v0.8.12/cheatsheet.html?highlight=reserved#reserved-keywords

---

Maybe replace(from, to) ?

Reply:
@devtooligan ^ ping